src/common/config.ts

```
@@ -158,6 +159,7 @@ const environmentConfig: EnvironmentConfigType = {
     OrgAdminGithubParentTeam: 14420860,
     GithubIdpSyncEnabled: false,
     AssetsBucketId: `427040638965-infra-core-api-assets-${genericConfig.AwsRegion}`,
+    AssetsBucketPublicUrl: 'core-assets.aws.qa.acmuiuc.org',
```

ESLint

[error] Replace `'core-assets.aws.qa.acmuiuc.org'` with `"core-assets.aws.qa.acmuiuc.org"`

(prettier/prettier)